src/diffpy/utils/scattering_objects/diffraction_objects.py

```diff
@@ -17,7 +17,7 @@
 )
 
 
-class Diffraction_object:
+class diffraction_object:
```

Comment:
Although this wasn't discussed before, could we try `CamelCase`? 

https://peps.python.org/pep-0008/#class-names

Reply:
Actually, not camel case which is not the PEP8 standard, but Classes always begin with capital letters.  Please everyone read the PEP8 standards....

Reply:
Seems it's got two interpretations:

<img width="320" alt="Screenshot 2024-11-07 at 4 30 11 PM" src="https://github.com/user-attachments/assets/3f860006-34b1-4e27-9bc4-5ac8f7b8c889">


while PEP8 says, 

> CapitalizedWords (or CapWords, or CamelCase – so named because of the bumpy look of its letters [[4]](https://peps.python.org/pep-0008/#id8)). This is also sometimes known as StudlyCaps.

but yeah I guess `CapWords` works..

Reply:
> Actually, not camel case which is not the PEP8 standard, but Classes always begin with capital letters. Please everyone read the PEP8 standards....

yeah I will add this to our gitlab so that we can say, "did you read that section" or "refer to here.."